src/Sylius/Bundle/ShippingBundle/Form/Type/ShippingMethodChoiceType.php

```diff
@@ -82,26 +82,22 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     public function configureOptions(OptionsResolver $resolver)
     {
         $choiceList = function (Options $options) {
-            $resolvedMethods = [];
+            $methods = $this->repository->findAll();
```

Comment:
We shouldn't query for all methods everytime. Let's use `if-else` or extract it to a separate function.